src/types.rs

```diff
@@ -487,6 +489,14 @@ impl From<&RuntimeConfig> for AppClientConfig {
 	}
 }
 
+#[derive(Serialize, Deserialize, PartialEq, Clone, Debug)]
```

Comment:
`Copy` derive is missing. I think clippy should warn for this

Reply:
You can also derive Default here and mark `Cells` as default variant

Reply:
There is no clippy warning, for what is `Copy` derive needed?

Reply:
Okay, there is a [rustc lint](https://doc.rust-lang.org/stable/nightly-rustc/rustc_lint/builtin/static.MISSING_COPY_IMPLEMENTATIONS.html) for that. We can enable it.

Copy is needed for declaring that some type can be fully allocated on stack. That enables a bunch of optimizations (basically copying `Copy` type will just use `memcpy` and deallocation of this thing on stack can also be skipped)